packages/core/lib/commands/install.js

```diff
@@ -1,35 +1,54 @@
+const { callbackify } = require("util");
+
 const command = {
   command: "install",
   description: "Install a package from the Ethereum Package Registry",
   builder: {},
   help: {
-    usage: "truffle install <package_name>[@<version>]",
+    usage: "truffle install <package-identifier> [--alias]",
     options: [
       {
-        option: "package_name",
-        description:
-          "Name of the package as listed in the Ethereum Package Registry. (required)"
+        option: "packageIdentifier",
+        description: `(required) Name of the package as listed in the Ethereum Package Registry. Accepted formats: packageName, packageName@version, ethpm URI, ipfs URI.`
```

Comment:
It is convention in Truffle to put the "(required)" at the end of the description.